pages/export.vue

```diff
@@ -114,7 +143,8 @@
                 @disable-button="(e) => (disableButton = e.disableButton)"
             />
             <div class="btn-container mt-3">
-                <button class="accent-btn" @click="download" :disabled="disableButton">Download Comic</button>
+                <button class="accent-btn" @click="download" :disabled="disableButton">Download as PNG</button>
+                <button class="accent-btn" @click="share" :disabled="disableButton">Share</button>
```

Comment:
I think this button should be styled as a secondary button to have more concistency with the other pages where only the main action button is full.

Despite this it is great!

Reply:
I discussed with Alica, made the changes and pushed, can you review again please?